static/components/TaskDetails.jsx

```diff
@@ -250,10 +246,10 @@ var TaskDetails = React.createClass({
           ref: task.sha,
         },
         success: (data) => {
-          let {app, environment, number} = this.props.task;
-          browserHistory.push(`/deploys/${app.name}/${environment}/${number}`);
-        }
-      });
+          browserHistory.push('/');
+          browserHistory.push(`/deploys/${data.app.name}/${data.environment}/${data.number}`);
```

Comment:
@JamesFTW – okay, can you link to that PR in a comment above this? Because I suspect very few people (see: me just now) will understand this at a glance.

Make that change and we're good to go.